translations/nl-BE.yaml

```diff
@@ -801,7 +798,15 @@ download:
     agendapoint-title: Titel agendapunt
     download-html: Download HTML
     no-data-message: Geen agendapunten gevonden in zitting
+    copy-options: Bekijk kopieermogelijkehden
 
 download-meeting-part:
   downloading: Aan het downloaden
   downloaded: Gedownload
+
+copy-options:
+  heading: 'Kopieer onderdelen uit: {besluit}'
```

Comment:
the dutch version has a format param, the english one doesn't, which throws when you switch to dutch

Reply:
worded that a bit weird, but its just that it expects a param in dutch which you're not giving

Reply:
Oh, I forgot to remove the format param from the dutch when I decided to highlight the title in the same way as the heading of the download page, so had to move it out.